it_IT/code-of-conduct.md

```diff
@@ -0,0 +1,30 @@
+___
```

Comment:
There is already a translated version : http://contributor-covenant.org/i18n/ (See link *Italiano* under section **Markdown**). It will just replace `[INSERIRE INDIRIZZO EMAIL]` in the file by `[sindresorhus@gmail.com](mailto:sindresorhus@gmail.com)`